CHANGELOG.rst

```diff
@@ -7,6 +7,193 @@ Changelog
     The Dutch version of this changelog can be found :ref:`here <changelog-nl>`.
 
 
+3.2.0 "Zomer" (2025-07-11)
+==========================
+
+Open Forms 3.2.0 is a feature release.
+
+.. epigraph::
+
+    "Zomer" is Dutch for "Summer".
+
+This contains the changes from the alpha releases and fixes applied until the stable version.
+BEFORE upgrading to 3.2.0, please read the release notes carefully and review the following
+instructions.
+
+Upgrade procedure
+-----------------
+
+# TODO-5398: which version?
+⚠️ Before upgrading to the 3.2 release series, please ensure you upgrade to Open Forms x.y.z first.
+
+.. warning::
+
+   The Camunda registration backend will be removed in Open Forms 4.0. There is no
+   replacement scheduled - if you rely on this plugin, please get in touch.
+
+.. warning::
+
+    For the Generic JSON registration plugin, we changed the way that the data is generated.
+    In case of key conflicts between static, component, and user-defined variables,
+    the static variables will take precedence. Previously, the component and user-defined
+    variables would override the static variables. Our validation guards against the use of
+    keys that are already present in the static variables, but this does not cover old forms
+    and newly-added static variables.
+
+Major features
+--------------
+
+**🔊 Improved logging**
+
+    We improved the logs that are emitted by the application, which enable better integration with observability
+    tooling like Grafana.
+
+    TODO-5398: should we mention that we also prototyped a dashboard? Can people even access this?
+
+**🔒️ Authentication using Yivi and eIDAS**
+
+    We implemented support for `Yivi <https://yivi.app/>`_ and `eIDAS <https://en.wikipedia.org/wiki/EIDAS>`_
+    authentication through OIDC. With support for Yivi authentication, end-users can decide which personal
```

Comment:
```suggestion
    authentication via the OpenID Connect protocol. With support for Yivi authentication, end-users can decide which personal
```
OIDC is not a very familiar abbreviation. It should be mentioned though, because we *do* have eIDAS via SAML too.